ocean_lib/ocean/test/test_ocean_assets.py

```diff
@@ -239,6 +239,7 @@ def test_ocean_assets_search(publisher_ocean, publisher_wallet):
     )
 
 
+@pytest.mark.skip(reason="Skip this until #1268 is merged.")
```

Comment:
@mariacarmina please fix inside this PR. All you need to do is cherry-pick 11ce25f09cf4873055740bf4eff17f49d3628671 and the subsequent changes here: https://github.com/oceanprotocol/ocean.py/pull/1265/files#diff-dfe25f756de0539701ed9f2643a7d93f22dfb4649eac3e4a6d7f0bd0820545c7